ors-api/src/main/java/org/heigit/ors/api/responses/snapping/json/JsonSnappingResponse.java

```diff
@@ -0,0 +1,37 @@
+package org.heigit.ors.api.responses.snapping.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.heigit.ors.api.responses.matrix.json.JSON2DSources;
+import org.heigit.ors.api.responses.matrix.json.JSONIndividualMatrixResponse;
+import org.heigit.ors.api.responses.matrix.json.JSONLocation;
+import org.heigit.ors.api.responses.snapping.SnappingResponse;
+import org.heigit.ors.matrix.ResolvedLocation;
+import org.heigit.ors.snapping.SnappingResult;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Schema(name = "SnappingResponse", description = "The Snapping Response contains the snapped coordinates.")
```

Comment:
Even though the response is quite simple it could make sense to have response examples for the docs api

Reply:
I can see one example being generated in the swagger ui which is probably derived from the JSONLocation type:
![grafik](https://github.com/GIScience/openrouteservice/assets/23240110/a9805f58-798f-4a0e-b766-b2c16347f6c9)
This is ok i think.